_docs/reference/config/kubernetes-proxy-injection.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Kubernetes sidecar proxy injection
+overview: Kubernetes sidecar proxy injection
+
+order: 15
+
+layout: docs
+type: markdown
+---
+
+Each pod in the Kubernetes cluster must be running an Istio compatible 
+sidecar. The following sections describe two ways of injecting the Istio 
+sidecar into a pod: manually using istioctl `kube-inject` or automatically 
+using the Istio sidecar injector webhook. 
+
+Manual injection modifies the controller configuration, e.g. deployment. It 
+does this by modifying the pod template spec such that *all* pods for that
+deployment are created with the injected sidecar. Adding/Updating/Removing
+the sidecar requires modifying the entire deployment.
+
+Automatic injection injects at pod creation time. The controller resource is 
+unmodified. Sidecars can be selectively added/updated/removed by (re)starting 
+pods after modifying the injection configuration.
+
+Manual and automatic injection use the same templated configuration. See TBD 
+for template configuration format.
+
+# Manual injection
+
+`kube-inject` is designed to be run offline without access to a running Kubernetes
+cluster. 
+
+Download a copy of the injection and mesh configmap. This might typically be retrieved 
+from a SCM as part of a CI/CD pipeline.
+
+```
+$ kubectl -n istio-system get configmap istio-inject -o=jsonpath='{.data.config}' > inject-config.yaml 
+$ kubectl -n istio-system get configmap istio -o=jsonpath='{.data.mesh}' > mesh-config.yaml
+```
```

Comment:
@ayj could we add steps to create the install/kubernetes/istio-sidecar-injector-configmap-release.yaml here as that is required to get the above 2 kubectl cmds working?